.gitignore

```diff
@@ -40,6 +40,9 @@ jspm_packages
 # Optional REPL history
 .node_repl_history
 
+# IDE config directories
```

Comment:
We don't include IDE paths for this reason: https://jaketrent.com/post/gitignore-system-files/

Reply:
So is the suggestion that it should be in your global .gitignore instead of in the repo?
In DevStar repos we actually do commit parts of the .idea folder (code style settings, run configs, live templates, etc...) so we don't have it to our global .gitignore.

Reply:
yeah, at the least, please don't include `.idea` in this project's gitignore.

Reply:
Any suggestions on how I can prevent those files from being committed here but not in the other repos I work in?

Reply:
If you don't want to use a system gitignore for this, I would just be more specific in what you `git add` w/in this project.

Reply:
Figured it out. I just added an entry to my global .gitignore for just the .idea folder in that repo.